harvester/mets.py

```diff
@@ -20,16 +20,18 @@ class METS:
     # This is expected to change when the software develops
     # pylint: disable=too-few-public-methods
 
-    def __init__(self, mets_path, binding_dc_identifier, encoding="utf-8"):
+    def __init__(self, binding_dc_identifier, mets_file, encoding="utf-8"):
         """
         Create a new METS file object.
 
+        :param binding_dc_identifier: DC identifier of binding
         :param mets_path: Path to the METS file.
```

Comment:
This is now outdated.

`mets_file` is now an open file-like object, right? We might run  into problems later if we have big bindings and do downloads for large images, the time the file and the underlying connection is open might be very long, which might manifest as timeouting. But that's a bridge we can cross later: the current handling for max couple of dozen ALTOs without fixity checking etc is not likely to cause problems.

Reply:
I fixed the outdated docstring. 

The `mets_file` is indeed an open file object, which gets closed after `files` are read from the METS. This was the only way I could come up with to handle remote and local METS files with the same logic.